lakefile.toml

```diff
@@ -9,7 +9,6 @@ relaxedAutoImplicit = false # switch off relaxed auto-implicit
 [[require]]
 name = "mathlib"
 git = "https://github.com/leanprover-community/mathlib4.git"
-rev = "master"
```

Comment:
Is this really right?

Reply:
Yes, the `rev` field it's unnecessary in our case. When we want to pin the `stable` version we can always add it.